README.md

```diff
@@ -91,13 +91,13 @@ specific RuboCop release.**
 
 **RuboCop**'s installation is pretty standard:
 
-```
+```sh
```

Comment:
So, what do you think to change to bash key highlight ? Or keep in this form? :D